CONTRIBUTING.md

```diff
@@ -77,3 +77,50 @@ The version in our package.json gets copied to the one we publish, and users nee
 ## ESLint parser
 
 As this workspace has a small amount of projects, we opt in to use the ESLint TypeScript parser globally (see the `parser` option in `.eslintrc.json`) in order to enable type-depending lint rules such as some from SonarLint.
+
+## Creating new applications
+
+In this workspace we have chosen to use the **npm nx preset** which has the following layout:
+
+```json
+  "workspaceLayout": {
+    "appsDir": "e2e",
+    "libsDir": "packages"
+  }
+```
+
+That means that our applications are created by default in the `e2e` folder and our libraries in the `packages` folder.
+
+However, this is semantically incorrect and we have also decided that our application will leave in the packages folder.
+
+When creating a new app there are some steps that need to be taken in order to bypass the default behavior of the **npm nx preset**.
+
+First, we need to modify the workspace layout inside the `nx.json` file from
+
+```json
+  "workspaceLayout": {
+    "appsDir": "e2e",
+    "libsDir": "packages"
+  }
+```
+
+To
+
+```json
+  "workspaceLayout": {
+    "appsDir": "packages",
+    "libsDir": "e2e"
+  }
+```
+
+Once that's completed we can create our application with the corresponding generator command.
+
+Finally we MUST revert the changes made to the `nx.json` file.
+
+I cases where the application generator create a companion e2e project, we need to move it to the `e2e` folder. This can be done using the `@nrwl/workspace:move` command.
```

Comment:
issue: Correct typo. Correct grammar. Improve decisiveness to increase the impact of the second sentence.
```suggestion
In cases where the application generator creates a companion e2e project, we need to move it to the `e2e` folder. This is done using the `@nrwl/workspace:move` generator.
```